runtime-modules/content/src/nft/mod.rs

```diff
@@ -265,10 +265,14 @@ impl<T: Config> Module<T> {
     pub(crate) fn ensure_new_pending_offer_available_to_proceed(
         nft: &Nft<T>,
         participant_account_id: &T::AccountId,
+        witness_price: Option<<T as balances::Config>::Balance>,
```

Comment:
in `buy_nft` (and related) I have used the term `price_commit` with the same semantics. If you think that witness_price is better please change that one also, so there is no confusion (since both params serve the same purpose)